src/hooks/useMediaQuery/__stories__/useMediaQuery.stories.mdx

```diff
@@ -0,0 +1,63 @@
+import { useRef, useCallback, useState } from "react";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import useMediaQuery from "..";
+import "../../__stories__/general-hooks-stories.scss";
+
+<Meta title="Hooks/useMediaQuery" />
+
+# useMediaQuery
+
+- [Overview](#overview)
+- [Arguments](#arguments)
+- [Feedback](#feedback)
+
+## Overview
+
+This hook helps maps if a media query is matched or not
+
+<Canvas>
+  <Story name="Single Rule">
+    {() => {
+      const [mediaQueryIsMatching] = useMediaQuery(["screen and (max-width: 1023px) and (min-width: 768px)"]);
+      return (
+        <div>
+          media query - "screen and (max-width: 1023px) and (min-width: 768px)" is matching:{" "}
+          {mediaQueryIsMatching ? "true" : "false"}
+        </div>
+      );
+    }}
+  </Story>
+</Canvas>
+
+<Canvas>
+  <Story name="Multiple Rules">
+    {() => {
+      const [screenSizeMediaQuery, preferDarkColorScheme] = useMediaQuery([
+        "screen and (max-width: 1280px) and (min-width: 768px)",
+        "prefers-color-scheme: dark"
+      ]);
+      return (
+        <div>
+          <div>
+            media query - "screen and (max-width: 1280px) and (min-width: 768px)" is matching:{" "}
+            {screenSizeMediaQuery ? "true" : "false"}
+          </div>
+          <div>
+            media query - prefers-color-scheme: dark is matching:{" "}
+            {preferDarkColorScheme ? "true" : "false"}
+          </div>
+        </div>
+      );
+    }}
+  </Story>
+</Canvas>
+
+## Arguments
+
+<FunctionArguments>
```

Comment:
I think it's critical to add here also a documentation of the return value of the hooks as well, like in other hooks (there is few examples for it in our storybook)